src/py42/modules/securitydata.py

```diff
@@ -59,7 +60,7 @@ def get_plan_security_events(
         min_timestamp=None,
         max_timestamp=None,
     ):
-        """Gets legacy, endpoint security events.
+        """Gets legacy endpoint monitoring events. This does not retrieve cloud events.
```

Comment:
since they return generators can we change the names of these to `get_all_plan_security_events` and `get_all_user_security_events`

Reply:
Done